src/js/tech/tech.js

```diff
@@ -312,18 +312,18 @@ class Tech extends Component {
    * @method emulateTextTracks
    */
   emulateTextTracks() {
+    let tracks = this.textTracks();
+    if (!tracks) {
+      return;
+    }
+
     if (!window['WebVTT'] && this.el().parentNode != null) {
       let script = document.createElement('script');
-      script.src = this.options_['vtt.js'] || '../node_modules/vtt.js/dist/vtt.js';
+      script.src = this.options_['vtt.js'] || 'https://raw.githubusercontent.com/mozilla/vtt.js/v0.12.1/dist/vtt.js';
```

Comment:
It may be a little... ugly... but we already have browserify-versionify [in use](https://github.com/videojs/video.js/blob/master/build/grunt.js#L255-L266). We could use that to replace the `node_modules` reference on build:

```js
['browserify-versionify', {
  placeholder: '../node_modules/vtt.js/dist/vtt.js',
  version: 'https://raw.githubusercontent.com/gkatsev/vtt.js/vjs-v0.12.1/dist/vtt.min.js'
}],
```